gen/suites/actor_creation/illegal_actor.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+
+	"github.com/filecoin-project/test-vectors/chaos"
+	. "github.com/filecoin-project/test-vectors/gen/builders"
+)
+
+func actorAbortWithSystemExitCode(i int) func(v *Builder) {
+	return func(v *Builder) {
+		v.Messages.SetDefaults(GasLimit(1_000_000_000), GasPremium(1), GasFeeCap(200))
+
+		alice := v.Actors.Account(address.SECP256K1, abi.NewTokenAmount(1_000_000_000_000))
+		v.CommitPreconditions()
+
+		code := big.NewInt(int64(i))
+
+		msg := v.Messages.Raw(alice.ID, chaos.Address, chaos.MethodAbortWithSystemExitCode, MustSerialize(&code), Nonce(0), Value(big.Zero()))
+		v.CommitApplies()
+
+		v.Assert.Equal(exitcode.ExitCode(i), msg.Result.ExitCode)
```

Comment:
These exit codes are not what we expect - it looks like the VM is propagating the error code that the actor aborts with (1-15)...